src/main/java/com/amazon/opendistroforelasticsearch/sql/rewriter/nestedfield/Identifier.java

```diff
@@ -60,7 +60,7 @@ String name() {
     }
 
     private int separatorIndex() {
```

Comment:
np: maybe rename to `lastSeparatorIndex` ?

Reply:
good point